docs/source/system-requirements.rst

```diff
@@ -0,0 +1,59 @@
+.. ******************************************************************************
+.. * Copyright 2021 Intel Corporation
+.. *
+.. * Licensed under the Apache License, Version 2.0 (the "License");
+.. * you may not use this file except in compliance with the License.
+.. * You may obtain a copy of the License at
+.. *
+.. *     http://www.apache.org/licenses/LICENSE-2.0
+.. *
+.. * Unless required by applicable law or agreed to in writing, software
+.. * distributed under the License is distributed on an "AS IS" BASIS,
+.. * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. * See the License for the specific language governing permissions and
+.. * limitations under the License.
+.. *******************************************************************************/
+
+System Requirements
+===================
+
+Hardware Requirements
+*********************
+
+.. tabs::
+
+   .. tab:: GPU
+
+      - 6th Gen Intel® Core™ processor or higher
+      - Intel® Iris® Plus Graphics
+      - Intel® Iris® Xe Graphics
+      - Intel® Iris® Xe Max Graphics
+      - Intel® Iris® Graphics
+      - Intel® Iris® Pro Graphics
+
+   .. tab:: CPU
+
+      - Intel Atom® Processors
+      - Intel® Core™ Processor Family
+      - Intel® Xeon® Processor Family
+      - Intel® Xeon® Scalable Performance Processor Family
+
+Software Requirements
+*********************
+
+- GCC* 7.x or higher or Intel® C++ Compiler 19.1 and later
+- Intel® oneAPI DPC++ Compiler latest release (for oneAPI DPC++ interfaces)
+- Intel® oneAPI Threading Building Blocks latest release (for the multi-threaded version of oneDAL)
+- OpenCL(TM) 1.2 or later
+- C/C++ Compiler with C++11 support (or C++14 support on Windows*)
+- Microsoft Visual Studio* (2019 and 2022 versions) needed only if using Visual Studio IDE for development
+
+Operating Systems:
+
+- 64-bit Linux*: Ubuntu* 18.04 or higher
+- 64-bit Windows* 10
```

Comment:
Suggest to just mention that library support 64-bit only and do not mention this for each build.

For windows - let's add Windows Server 2019  or higher.  
For Win 10 - + or higher as we support Win11 

On linux - Let's point that in theory we will support any linux with certain GLIBC/BINUTILS requirements , and mention that we are testing RHEL and UBUNTU.
@maria-Petrova @homksei  @vmeshche  can comment

Reply:
done except for this:

> On linux - Let's point that in theory we will support any linux with certain GLIBC/BINUTILS requirements , and mention that we are testing RHEL and UBUNTU.
@maria-Petrova @homksei @vmeshche can comment

not sure what I need to add

Reply:
thanks